File: app/src/org/commcare/connect/network/connect/ConnectApiEndpoints.java
package org.commcare.connect.network.connect;

public class ConnectApiEndpoints {
    public static final String connectOpportunitiesURL = "/api/opportunity/";
    public static final String connectStartLearningURL = "/users/start_learn_app/";
    public static final String connectLearnProgressURL = "/api/opportunity/{id}/learn_progress";
    public static final String connectClaimJobURL = "/api/opportunity/{id}/claim";
    public static final String connectDeliveriesURL = "/api/opportunity/{id}/delivery_progress";
    public static final String PAYMENT_CONFIRMAITONS = "/api/payment/confirm";
}

File: app/src/org/commcare/connect/network/connect/ConnectApiEndpoints.kt
package org.commcare.connect.network.connect

object ConnectApiEndpoints {
    const val OPPORTUNITIES_URL = "/api/opportunity/"
    const val START_LEARNING_URL = "/users/start_learn_app/"
    const val LEARN_PROGRESS_URL = "/api/opportunity/{id}/learn_progress"
    const val CLAIM_JOB_URL = "/api/opportunity/{id}/claim"
    const val DELIVERIES_URL = "/api/opportunity/{id}/delivery_progress"
    const val PAYMENT_CONFIRMAITONS = "/api/payment/confirm"
}
